components/atom/input/src/InputWrapper.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import InputLabel from './Label'
+
+const InputWrapper = (props) => {
+  const {name, label, children} = props
+  return (
+    <div className='sui-AtomInput'>
```

Comment:
Are you sure you need a div. By default a span is the wrapper with less layout effects.
If you take into account my previous comment you could event return the label element as a wrapper as you're not using the sui-AtomInput class....